inst/private/python_ipc_driver.m

```diff
@@ -44,7 +44,10 @@
   end
 
   if (strcmp(lower(which_ipc), 'default'))
-    if (exist('popen2') > 1)
+    # TODO: may need to adjust, ideally just 'py'
```

Comment:
Yeah, in the future it would be really interesting to see if OctSymPy can work with Matlab's built in Python support using the same interface

Reply:
Yes, that would be nice.  I'm a bit hesitant to do that kind of work myself.